crates/net/network/src/subprotocol.rs

```diff
@@ -17,9 +17,9 @@ use std::{
 
 /// A trait that allows to offer additional RLPx-based application-level protocols when establishing
 /// a peer-to-peer connection.
-pub trait ProtocolHandler: fmt::Debug + Send + Sync + 'static {
+pub trait SubProtocolHandler: fmt::Debug + Send + Sync + 'static {
```

Comment:
one recommendation for when you're prototyping things

try to avoid renames on a feature like this because just increases the risk of conflicts and makes reviewing the actual feature a bit harder.
so we can undo the rename for name  and instead rename the eth specific protocolhandler to ethprotocolhandler